content/developer/tutorials/server_framework_101/05_connect_the_dots.rst

```diff
@@ -0,0 +1,449 @@
+              </list>
+          [...]
+      </record>
+
+      <record id="real_estate.property_form" model="ir.ui.view">
+          [...]
+              <group string="Listing Information">
+                  <field name="type_id"/>
+                  <field name="selling_price"/>
+                  <field name="best_offer_amount"/>
+                  <field name="availability_date"/>
+                  <field name="active"/>
+              </group>
+              <group string="Building Specifications">
+                  [...]
+                  <field name="garden_area"/>
+                  <field name="total_area"/>
+                  [...]
+              </group>
+          [...]
+      </record>
+
+.. _tutorials/server_framework_101/inverse_methods:
+
+Make computed fields editable
+-----------------------------
+
+You might have noticed that computed fields are read-only by default. This is expected since their
+values are typically determined programmatically rather than set manually by users. However, this
+behavior can be limiting when users need to adjust the computed value themselves. **Inverse
+methods** address this limitation by allowing edits to computed fields and propagating the changes
+back to their dependent fields.
+
+To make a computed field editable, a Python method must be defined and linked to the field
+declaration using the `inverse` argument. This method specifies how updates to the computed field
+should be applied to its dependencies.
+
+.. seealso::
+   :ref:`Reference documentation for related fields <reference/fields/related>`
+
+.. todo: inverse: offer deadline
+
+.. _tutorials/server_framework_101/store_computed_fields:
+
+Store computed fields
+---------------------
+
+As computed fields are calculated on the fly, recalculating their values repeatedly can become
+inefficient, especially when they are frequently accessed or used in models with large datasets.
+Another consequence is that they cannot be used in search queries by default. **Stored computed
+fields** address both these issues by saving their values in the database and automatically updating
+them only when their dependent data changes. Storing a computed field also enables the database to
+index the field's column, significantly improving query performance for large datasets.
+
+Computed fields can be stored in the database by including the `store=True` argument in their field
+declaration. The :code:`@api.depends()` decorator ensures that computed fields remain consistent not
+only in the cache, but also when they are stored in the database.
+
+However, storing computed fields should be carefully considered. Every update to a dependency
+triggers a recomputation, which can significantly impact performance on production servers with a
+large number of records.
+
+.. example::
+   store `margin`
+
+.. seealso::
+   Reference documentation for the :meth:`@api.depends() <odoo.api.depends>` decorator
```

Comment:
Nice catch! `depends` was originally introduced here but I had to move it earlier to not have to force the user to save the form so that the compute can triggers.